### ISSUE TITLE
Error: Attribute enterkeyhint not allowed on element input at this point.

### ISSUE THREAD

**Issue by Lin2Jing4** [validator/validator#1126]
code to reproduce error:
```html
<!DOCTYPE html>
<html lang="">
    <head>
        <title>Test</title>
    </head>
    <body>
        <input enterkeyhint="go">
    </body>
</html>
```

### Steps to reproduce
1. Navigate to http://validator.w3.org/nu/#textarea .
2. Replace `<p></p>` with `<input enterkeyhint="go">`.
3. Click `Check`.

### Actual behavior
> **Error:** Attribute `enterkeyhint` not allowed on element [`input`](https://html.spec.whatwg.org/multipage/input.html#the-input-element) at this point.
> From line 7, column 1; to line 7, column 25
> `d>↩<body>↩<input enterkeyhint="go">↩</bod`
> Attributes for element input:
> [Global attributes](https://html.spec.whatwg.org/multipage/dom.html#global-attributes)

### Expected behavior
No errors or warnings to show since the HTML Living Standard says: (emphasis mine)
> [§ 3.2.6 Global attributes](https://html.spec.whatwg.org/multipage/dom.html#global-attributes)
> The following attributes are common to and may be specified **on all HTML elements** (even those not defined in this specification):
> 
> - accesskey
> - autocapitalize
> - autofocus
> - contenteditable
> - dir
> - draggable
> - [enterkeyhint](https://html.spec.whatwg.org/multipage/interaction.html#input-modalities:-the-enterkeyhint-attribute)
